tests/pkg_micro-ecc-with-hwrng/tests/01-run.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import math
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+
+def testfunc(child):
+    child.expect_exact('micro-ecc compiled!')
+    child.expect_exact('Testing 16 random private key pairs and signature '
+                       'using HWRNG')
+    child.expect_exact('................ done with 0 error(s)')
+    child.expect('SUCCESS')
```

Comment:
theoretically this would be enough to check/wait for, btw. should `expect_exact` I guess